### DISCUSSION TITLE
Scaffolded exploration for the span of two vectors?

### DISCUSSION THREAD

**Discussion by StevenClontz** [TeamBasedInquiryLearning/linear-algebra#247] [Ideas]
Students do pretty well with figuring out the span of one vector is a line: [Activity 2.1.5](https://teambasedinquirylearning.github.io/linear-algebra/2023/EV1.html#activity-27) 

![image](https://github.com/TeamBasedInquiryLearning/linear-algebra/assets/1559632/ae7fc3e4-8293-45ac-bd58-8a5a428458c4)

But I got a *lot* of parabola votes from this exploration: [Activity 2.1.6](https://teambasedinquirylearning.github.io/linear-algebra/2023/EV1.html#activity-28) 

![image](https://github.com/TeamBasedInquiryLearning/linear-algebra/assets/1559632/81e6b15b-e454-49b6-bdb4-c3d6414c8154)

Any pitches for how to maybe better scaffold this exploration so students see the planar geometry a bit better?

(For whatever it's worth, they did all decide the next activity drew out a line: [Activity 2.1.7](https://teambasedinquirylearning.github.io/linear-algebra/2023/EV1.html#activity-29))

![image](https://github.com/TeamBasedInquiryLearning/linear-algebra/assets/1559632/e95d9744-f3f1-4e5a-9c34-dd0dc83b1e07)

**Comment by siwelwerd** [comment]
Not sure how to improve it at the moment, but I also typically get a lot of parabola votes.

The IOLA materials have a nice "magic carpet" task sequence that might have some ideas for us: 
https://iola.math.vt.edu/media/unit1/docs/u1t1.pdf
https://doi.org/10.1080/10511970.2012.667516

**Reply by StevenClontz** [reply]
One thing I think we might be missing is a visualization of vector addition to begin with.

Or maybe this is another opportunity for an interactive to save the day, rather than having students spend a lot of time computing linear combos to try out.

PS: am I a bad teacher, or am I justified in looking through that facilitation guide and thinking "who the heck reads all of that"?

**Comment by StevenClontz** [comment]
Here's a Doenet.org interactive (a bit slow, sadly): https://www.doenet.org/portfolioviewer/_gDhVzoC33ojk4E28hTWmQ
